### PR TITLE
Add support for loading kafka properties from classpath

### PR DESCRIPTION
**Pull request by alvinkwekel**
We have the requirement to include the kafka properties from the classpath

### CODE REVIEW

**Comment by Bert-R** [comment]
Closing and reopening to retrigger the checks

File: src/main/java/kafdrop/config/KafkaConfiguration.java [hunk 1]
```diff
@@ -48,12 +52,16 @@ public void applyCommon(Properties properties) {
     }
 
     LOG.info("Checking properties file {}", propertiesFile);
-    final var propertiesFile = new File(this.propertiesFile);
-    if (propertiesFile.isFile()) {
-      LOG.info("Loading properties from {}", this.propertiesFile);
+    Optional<AbstractResource> readablePropertyFile = StringUtils.isBlank(propertiesFile) ? Optional.empty() :
```
Comment (Bert-R) [inline src/main/java/kafdrop/config/KafkaConfiguration.java]:
```suggestion
    Optional<AbstractResource> propertiesResource = StringUtils.isBlank(propertiesFile) ? Optional.empty() :
```

**Review by Bert-R — CHANGES_REQUESTED** [review]
@alvinkwekel Thanks for this enhancement!
Code looks good, except for the name of the variable holding the properties resource. If you rename that, we can merge this PR.

File: src/main/java/kafdrop/config/KafkaConfiguration.java (hunk 1 above)
Comment (alvinkwekel) [reply src/main/java/kafdrop/config/KafkaConfiguration.java]: @Bert-R Thanks for the feedback. I've changed the variable name.

**Review by Bert-R — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**